.storybook/main.js

```diff
@@ -43,22 +44,22 @@ module.exports = {
       require.resolve('babel-plugin-remove-graphql-queries'),
     ];
 
-    // TODO: Figure out why Gatsby is throwing this error:
-    // 'The result of this StaticQuery could not be fetched' & remove this alias.
-    ['.', '..', '../..', '../../..'].forEach((pathPrefix) => {
-      config.resolve.alias = {
-        ...config.resolve.alias,
-        [`${pathPrefix}/lib/useSiteMetadata`]: p.resolve(__dirname, './useSiteMetadata.js'),
-      };
-    });
-
     config.plugins.unshift(
       // eslint-disable-next-line new-cap
       new CoreJSUpgradeWebpackPlugin.default({
         resolveFrom: [coreJsLocationOfRoot, coreJsLocationOfGatsby],
       })
     );
 
+    // TODO: Figure out why Gatsby is throwing this error:
+    // 'The result of this StaticQuery could not be fetched' & remove this alias.
+    config.plugins.unshift(
+      new webpack.NormalModuleReplacementPlugin(
+        /lib\/useSiteMetadata\.js/,
+        '../../../.storybook/useSiteMetadata.js'
```

Comment:
Maybe use `require.resolve('./useSiteMetadata.js')` rather than this kind of hard to understand path?